### PR TITLE
Updating role/collection references

### PR DESCRIPTION
**Pull request by ricrodriguezg**

### CODE REVIEW

**Review by rvelaVenafi — APPROVED** [review]

**Review by angelmoo — APPROVED** [review]

**Review by luispresuelVenafi — APPROVED** [review]

**Review by luispresuelVenafi — APPROVED** [review]
LGTM +1

File: roles/certificate/examples/citrix_adc/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Configuring secure application delivery using Citrix ADC and the Venafi Ansible Role
+# Configuring secure application delivery using Citrix ADC and the Venafi Ansible Collection
 
-In this example, we'll show you how to better secure application delivery using the Venafi Ansible Role with your [Citrix ADC](https://www.citrix.com/products/citrix-adc/) instance.
+In this example, we'll show you how to better secure application delivery using the Venafi Ansible Collection with your [Citrix ADC](https://www.citrix.com/products/citrix-adc/) instance.
```
Comment (rvelaVenafi) [inline roles/certificate/examples/citrix_adc/README.md:3]: Venafi Collection for Ansible

File: roles/certificate/examples/citrix_adc/README.md [hunk 2]
```diff
@@ -11,9 +11,9 @@ The steps described in this document are typically performed by _DevOps engineer
...
 
-In this example, we use [RedHat Ansible](https://www.ansible.com/) with the _Venafi Ansible Role_ to automate the process of requesting, retrieving and installing a certificate as part of SSL termination on an ADC (specifically, Citrix ADC) for load balancing web traffic. We'll also utilize three HTTP servers contained in a cluster as the endpoints that are sending and receiving web traffic and being managed by Citrix ADC.
+In this example, we use [RedHat Ansible](https://www.ansible.com/) with the _Venafi Ansible Collection_ to automate the process of requesting, retrieving and installing a certificate as part of SSL termination on an ADC (specifically, Citrix ADC) for load balancing web traffic. We'll also utilize three HTTP servers contained in a cluster as the endpoints that are sending and receiving web traffic and being managed by Citrix ADC.
```
Comment (rvelaVenafi) [inline roles/certificate/examples/citrix_adc/README.md:14]: Venafi Collection for Ansible

File: roles/certificate/examples/citrix_adc/README.md [hunk 3]
```diff
@@ -11,9 +11,9 @@ The steps described in this document are typically performed by _DevOps engineer
...
 
-Later in this example, you'll generate a certificate for the `demo-citrix.venafi.example` domain using the Venafi Ansible Role to request and retrieve it from either _Venafi Trust Protection Platform_ or _Venafi Cloud_ services. Then you'll copy the certificate files (certificate, private key, chain Bundle) to the Citrix ADC. Finally, you'll configure Citrix ADC to distribute the traffic between three NGINX servers using the round-robin load balancing method. Here below you can find a diagram of what we are trying to accomplish.
+Later in this example, you'll generate a certificate for the `demo-citrix.venafi.example` domain using the Venafi Ansible Collection to request and retrieve it from either _Venafi Trust Protection Platform_ or _Venafi Cloud_ services. Then you'll copy the certificate files (certificate, private key, chain Bundle) to the Citrix ADC. Finally, you'll configure Citrix ADC to distribute the traffic between three NGINX servers using the round-robin load balancing method. Here below you can find a diagram of what we are trying to accomplish.
```
Comment (rvelaVenafi) [inline roles/certificate/examples/citrix_adc/README.md:16]: same issue

File: roles/certificate/examples/citrix_adc/README.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ Later in this example, you'll generate a certificate for the `demo-citrix.venafi
 
 To perform the tasks described in this example, you'll need:
 
-- The Venafi Ansible Role installed on your machine; you can install it using `ansible-galaxy` [as described here](https://github.com/Venafi/ansible-role-venafi#using-with-ansible-galaxy)
+- The Venafi Ansible Collection installed on your machine; you can install it using `ansible-galaxy` [as described here](https://github.com/Venafi/ansible-role-venafi#using-with-ansible-galaxy)
```
Comment (rvelaVenafi) [inline roles/certificate/examples/citrix_adc/README.md:28]: same issue

File: roles/certificate/examples/citrix_adc/README.md [hunk 5]
```diff
@@ -37,14 +37,14 @@ To perform the tasks described in this example, you'll need:
 
 Here are the steps we'll complete as we go through this example:
 
-1. Retrieve a certificate using the Venafi Ansible Role
+1. Retrieve a certificate using the Venafi Ansible Collection
```
Comment (rvelaVenafi) [inline roles/certificate/examples/citrix_adc/README.md:40]: same issue

File: roles/certificate/examples/citrix_adc/README.md [hunk 6]
```diff
@@ -37,14 +37,14 @@ To perform the tasks described in this example, you'll need:
...
 2. Copy the retrieved certificate files to Citrix ADC
 3. Create a certificate-key pair on Citrix ADC 
 4. Create HTTP back-end services on Citrix ADC
 5. Create a virtual server on Citrix ADC
 6. Execute the playbook
 
-## Step 1: Retrieve a certificate using the Venafi Ansible Role
+## Step 1: Retrieve a certificate using the Venafi Ansible Collection
```
Comment (rvelaVenafi) [inline roles/certificate/examples/citrix_adc/README.md:47]: same issue

File: roles/certificate/examples/citrix_adc/README.md [hunk 7]
```diff
@@ -82,13 +82,15 @@ chain_name: "{{ test_site.name }}-ca-bundle.crt"
 
 ### Step 1b: Creating the playbook
 
-Start by creating a YAML file named `citrix_create_playbook.yaml`, inside, define a name for the playbook, the hosts in which the tasks will be executed, the type of connection to use, the Citrix ADC collection and specify the variables file created in the previous step :
+Start by creating a YAML file named `citrix_create_playbook.yaml`, inside, define a name for the playbook, the hosts in which the tasks will be executed, the type of connection to use, the Citrix ADC collection as well as the Venafi Ansible Collection and specify the variables file created in the previous step :
```
Comment (rvelaVenafi) [inline roles/certificate/examples/citrix_adc/README.md:85]: same issue

File: roles/certificate/examples/citrix_adc/README.md [hunk 8]
```diff
@@ -97,7 +99,7 @@ Start by creating a YAML file named `citrix_create_playbook.yaml`, inside, defin
 
 ### Step 1c: Requesting and retrieving the certificate using Venafi Role
 
-In the following block of instructions the Venafi Ansible Role is being specified along with the variables it needs to request and retrieve the certificate from the Venafi services, by adding these instructions the Ansible Role will:
+In the following block of instructions the Certificate Role included in the Venafi Ansible Collection is being specified along with the variables it needs to request and retrieve the certificate from the Venafi services, by adding these instructions Certificate Role will:
```
Comment (rvelaVenafi) [inline roles/certificate/examples/citrix_adc/README.md:102]: same issue

File: roles/certificate/examples/f5_bigip/README.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
-# Configuring secure application delivery using F5 BIG-IP and the Venafi Ansible Role
+# Configuring secure application delivery using F5 BIG-IP and the Venafi Ansible Collection
```
Comment (rvelaVenafi) [inline roles/certificate/examples/f5_bigip/README.md:1]: same issue here

**Review by marcos-albornoz — APPROVED** [review]

**Pull request merged**